docs/af-fundamentals-tutorial/modules/deadlines/pages/implementing.adoc

```diff
@@ -0,0 +1,77 @@
+= Implementing Deadlines
+:navtitle: Add a handler
+
+As the `SimpleDeadlineManager` is included in the messaging module of Axon framework, and uses Java's `ScheduledExecutorService` no additional dependencies are needed to use it.
+The `SimpleDeadlineManager` can be added both as an Axon Configuration component, or as a Spring Bean.
+For this howto we will configure it as a Spring Bean.
+
+[source,java]
+----
+@Bean
+public DeadlineManager deadlineManager(
+        org.axonframework.config.Configuration configuration
+) {
+    var provider = new ConfigurationScopeAwareProvider(configuration);
+    return SimpleDeadlineManager
+            .builder()
+            .scopeAwareProvider(provider)
+            .build();
+}
+----
+
+There are some additional things that could be set.
```

Comment:
What do you mean here with 'things'? I would rather refer to it as configuration or something